core/src/test/java/org/apache/accumulo/core/clientImpl/TableOperationsHelperTest.java

```diff
@@ -133,22 +133,18 @@ public void flush(String tableName, Text start, Text end, boolean wait) {}
 
     @Override
     public void setProperty(String tableName, String property, String value) {
-      if (!settings.containsKey(tableName))
-        settings.put(tableName, new TreeMap<>());
+      settings.putIfAbsent(tableName, new TreeMap<>());
       settings.get(tableName).put(property, value);
     }
 
     @Override
     public Map<String,String> modifyProperties(String tableName,
         Consumer<Map<String,String>> mapMutator)
         throws IllegalArgumentException, ConcurrentModificationException {
+      settings.putIfAbsent(tableName, new TreeMap<>());
```

Comment:
```suggestion
      settings.computeIfAbsent(tableName, k -> new TreeMap<>());
```

Reply:
Small nit but if you use computeIfAbsent here then you save on the allocation each time of a new TreeMap that you don't need unless the value is absent so you reduce garbage collection since it's lazy allocated. Not a big deal since this is just a unit test but in general useful.

Reply:
Oh cool, I did not know that, thanks!